server/api/config.go

```diff
@@ -6,22 +6,31 @@ import (
 
 // StoredConfig represents the data stored in and managed with the Mattermost
 // config.
+//
+// StoredConfig should be abbreviated as sc.
 type StoredConfig struct {
-	Apps               map[string]interface{}
-	AWSAccessKeyID     string
-	AWSSecretAccessKey string
-}
+	// InstalledApps is a list of all apps installed on the Mattermost instance.
+	//
+	// For each installed app, an entry of string(AppID) -> sha1(App) is added,
+	// and the App struct is stored in KV under app_<sha1(App)>. Implemenrtation
+	// in `store.App`.
+	InstalledApps map[string]string `json:"installed_apps,omitempty"`
 
-type ConfigMapper interface {
-	ConfigMap() (result map[string]interface{})
-}
+	// LocalManifests is a list of locally-stored manifests. Local is in
+	// contrast to the "global" list of manifests which in the initial version
+	// is loaded from S3.
```

Comment:
I guess there is the corner case where the global list contains a pointer to a file:// manifest which can be a local one - local server/aws deployment would be another contrived example ?)

Reply:
The case of having a `file://` listed in `manifests.json` is hypothetical, and is intended mostly for internal debugging. 

The locally-installed manifests are those explicitly added - for now dev mode, eventually to add unlisted (custom/3rd party) Apps. One _could_ just edit the manifests file, but this is an override mechanism.